inst/css/filter-panel.css

```diff
@@ -252,6 +252,18 @@ a.remove_all:hover {
   margin: 0em;
 }
 
+.filter-card:has(> .filter-card-header[aria-expanded="true"]) {
+  transition-property: background-color;
+  transition-duration: 150ms;
+  background-color: #ffffffb8;
+}
+
+.filter-card:has(> .filter-card-header[aria-expanded="true"]):hover {
+  transition-property: box-shadow;
+  transition-duration: 150ms;
+  box-shadow: 0 0 5px 2px #00000033;
+}
```

Comment:
The box-shadow is distracting. Even without the shadow, the change in the background color is enough to distinguish between the selected and unselected filter.
<img width="426" alt="Screenshot 2024-03-11 at 12 04 15 PM" src="https://github.com/insightsengineering/teal.slice/assets/49812166/eaa843b4-c5e3-4208-a455-ffd8de2da690">